### PR TITLE
qsopt 1.01 (new formula)

### PR DESCRIPTION
**Pull request by Moisan**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [x] Written a sensible test? (The best test is to compile and run a sample program.)

A LP solver from the university of Waterloo.

I have doubt on my use of wget to obtain missing files. Is there any cleaner way to do this?

### CODE REVIEW

File: qsopt.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  version "1.01"
+  sha256 "fa2f135eb6fa14e4c27abc97549ab032f56155170bbe27d2c0094245fe55e60e"
+
+  depends_on "wget"
+
+  def install
+    bin.install "qsopt"
+    system "wget", "http://www.math.uwaterloo.ca/~bico/qsopt/beta/codes/mac64/qsopt.h"
+    include.install "qsopt.h"
+    system "wget", "http://www.math.uwaterloo.ca/~bico/qsopt/beta/codes/mac64/qsopt.a"
```
Comment (dpo) [inline qsopt.rb]: Is the source code not available? Under what license is this distributed?

**Comment by Moisan** [comment]
The source code is not available and the only mention of copyright I found is this:

> all rights to QSopt are maintained by the authors, David Applegate, William Cook, Sanjeeb Dash, and Monika Mevenkamp.

Is it problematic for redistribution via homebrew?

**Comment by dpo** [comment]
Not necessarily but it's annoying that the licensing terms aren't clear.

**Comment by dpo** [comment]
The `.h` and `.a` should be resources I think. You would eliminate the dependency on `wget.

**Comment by dpo** [comment]
Thanks!

**Pull request closed without merging**